lib/core/src/examples/css3d_system.rs

```diff
@@ -0,0 +1,87 @@
+#![allow(missing_docs)]
+
+use crate::system::web::dom::html::Css3dSystem;
+use crate::system::web::dom::html::Css3dObject;
+use crate::system::web::dom::html::Css3dPosition;
+use crate::system::web::StyleSetter;
+use crate::display::object::DisplayObject;
+use crate::display::object::DisplayObjectOps;
+use crate::display::symbol::geometry::Sprite;
+use crate::display::symbol::geometry::SpriteSystem;
+use crate::display::world::*;
+use crate::prelude::*;
+use crate::system::web::forward_panic_hook_to_console;
+use crate::system::web::set_stdout;
+use crate::animation::animator::fixed_step::FixedStepAnimator;
+
+use nalgebra::Vector2;
+use nalgebra::Vector3;
+use wasm_bindgen::prelude::*;
+use crate::display::navigation::navigator::Navigator;
+
+#[wasm_bindgen]
+#[allow(dead_code)]
+pub fn run_example_css3d_system() {
+    forward_panic_hook_to_console();
+    set_stdout();
+    init(WorldData::new("canvas"));
+}
+
+fn init(world:World) {
+    let scene         = world.scene();
+    let camera        = scene.camera();
+    let screen        = camera.screen();
+    let navigator     = Navigator::new(&scene, &camera).expect("Couldn't create navigator");
+    let sprite_system = SpriteSystem::new(&world);
+    let css3d_system  = Css3dSystem::new(&world);
+    world.add_child(&sprite_system);
+    world.add_child(&css3d_system);
+
+    let mut sprites: Vec<Sprite> = default();
+    let mut css3d_objects: Vec<Css3dObject> = default();
+    let count = 10;
+    for i in 0 .. count {
+        let x          = i as f32;
+        let width      = screen.width * 1.5 / count as f32;
+        let height     = screen.height;
+        if i % 2 == 0 {
+            let height = height * 0.75;
+            let dimensions = Vector2::new(width, height);
+            let y = screen.height / 2.0;
+            let position   = Vector3::new(width / 1.5 * x + width / 2.0, y, 0.0);
+            let sprite = sprite_system.new_instance();
+            sprite.size().set(dimensions);
+            sprite.mod_position(|t| *t = position);
+            sprites.push(sprite);
+        } else {
+            let dimensions = Vector2::new(width, height);
+            let position   = Vector3::new(width / 1.5 * x + width / 2.0, height / 2.0, 0.0);
+            let mut object = css3d_system.new_instance("div").expect("Couldn't create div");
+            let r = ((x + 0.0) * 16.0) as u8;
+            let g = ((x + 2.0) * 32.0) as u8;
+            let b = ((x + 4.0) * 64.0) as u8;
+            let color = format!("rgb({},{},{})", r,g,b);
+            object.dom().set_property_or_panic("background-color", color);
+            object.set_dimensions(dimensions);
+            object.mod_position(|t| *t = position);
+            css3d_objects.push(object);
+        }
+    }
+    world.display_object().update();
```

Comment:
This should not be needed. If its needed, its my bug.

Reply:
Yup, it's needed. It should be called before render calls, right?